cli/src/main/kotlin/commands/RequirementsCommand.kt

```diff
@@ -100,28 +103,66 @@ class RequirementsCommand : CliktCommand(name = "requirements", help = "List the
             }
         }
 
+        var statusCode = 0
+
         allTools.forEach { (category, tools) ->
             println("${category}s:")
+
             tools.forEach { tool ->
-                // TODO: State which version was found, and whether it could be bootstrapped, but that requires
-                //       refactoring of CommandLineTool.
+                // TODO: State whether a tool can be bootstrapped, but that requires refactoring of CommandLineTool.
                 val message = buildString {
-                    if (tool.isInPath()) append("\t* ") else append("\t- ")
+                    val (prefix, suffix) = if (tool.isInPath()) {
+                        try {
+                            val actualVersion = tool.getVersion()
+                            try {
+                                if (tool.getVersionRequirement().isSatisfiedBy(actualVersion)) {
+                                    Pair("\t* ", "Found version $actualVersion.")
+                                } else {
+                                    statusCode = statusCode.or(2)
+                                    Pair("\t+ ", "Found version $actualVersion.")
```

Comment:
Consider adding ` (but '${tool.getVersionRequirement}' is required)` to the string.

Reply:
That's basically already done in line 148.

Reply:
FYI, an excerpt of the new output looks like:

```
...
PackageManagers:
        * Bower: Requires 'bower' in version =1.8.8. Found version 1.8.8.
        * Bundler: Requires 'bundle' in version >=1.16.0 <2.2.0. Found version 2.0.2.
        * Cargo: Requires 'cargo' in no specific version. Found version 1.39.0.
        * Conan: Requires 'conan' in version =1.18.0. Found version 1.18.0.
        + GoDep: Requires 'dep' in no specific version. Found version 'devel'.
        * GoMod: Requires 'go' in no specific version. Found version 1.10.4.
        * Npm: Requires 'npm' in version >=5.7.0 <6.15.0. Found version 6.9.0.
        - PhpComposer: Requires 'composer' in version >=1.5.0. Tool not found.
        * Pip: Requires 'pip' in no specific version. Found version 18.1.
        - Pipenv: Requires 'pipenv' in version >=2018.10.9. Tool not found.
        * Pub: Requires 'pub' in version >=2.2.0. Found version 2.4.0.
        + Sbt: Requires 'sbt' in version >=0.13.0. Could not determine the version.
        * Stack: Requires 'stack' in version >=2.1.1. Found version 2.1.1.
        * Yarn: Requires 'yarn' in version >=1.3.0 <1.23.0. Found version 1.16.0.
...
```